commit 14ba95d61c68988b6854f917cceb04fadf3f5a52 (refs/changes/81/4781/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-09-13T16:49:57+02:00 (1 year, 1 month ago)
    
    TRITON-685 cn-agent needs to support image.requirements.bootrom
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/package.json b/package.json
index 0076775..7b739d4 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "cueball": "2.7.1",
     "digest-stream": "0.2.2",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
-    "imgmanifest": "3.0.0",
+    "imgmanifest": "3.1.0",
     "jsprim": "2.0.0",
     "kstat": "1.0.1",
     "lazy-property": "1.0.0",
